### ISSUE TITLE
Tetrahedra mesh demo does not work with the newer version of Py3DViewer

### ISSUE THREAD

**Issue by vsnever** [raysect/source#374]
After [recent refactoring](https://github.com/cg3hci/py3DViewer/commit/dc315b8236e752ac7bfc712e343a6e62d7e31066) of Py3DViewer module, the `tets` and `num_tets` attributes of `Tetmesh` class were renamed to `polys` and `num_polys`, so the [tetrahedra_mesh_emission.py](https://github.com/raysect/source/blob/ea529db8804509e7e13aab25dd03a22b22c890b4/demos/materials/tetrahedra_mesh_emission.py) demo now throws an AttributeError at:
```python
tetra = Discrete3DMesh(mesh.vertices, mesh.tets, np.ones((mesh.num_tets)), False, 0)
```
For backward compatibility, this should be modified to:
```python
try:
    tetra = Discrete3DMesh(mesh.vertices, mesh.polys, np.ones((mesh.num_polys)), False, 0)
except AttributeError:
    tetra = Discrete3DMesh(mesh.vertices, mesh.tets, np.ones((mesh.num_tets)), False, 0)
```

**Comment by CnlPepper** [comment]
Please open a PR with the relevant changes.

**Comment by vsnever** [comment]
Fixed in #377.
